arthur_bench/scoring/scorer.py

```diff
@@ -27,6 +27,17 @@ def _can_omit(parameter: Parameter):
     return is_optional or is_kwargs
 
 
+class Feedback:
```

Comment:
you can use python [dataclasses](https://docs.python.org/3/library/dataclasses.html) to get the `__init__` and other base functionality for free